### PR TITLE
codec: add more datum compare tests.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

File: src/util/codec/datum.rs [hunk 1]
```diff
@@ -353,7 +355,13 @@ mod test {
     fn test_datum_cmp() {
         let tests = vec![
             (Datum::F64(1.0), Datum::F64(1.0), Ordering::Equal),
+            (Datum::F64(1.0), Datum::F64(f64::MAX), Ordering::Less),
+            (Datum::F64(1.0), Datum::F64(f64::MIN), Ordering::Greater),
+            (Datum::F64(1.0), Datum::F64(f64::INFINITY), Ordering::Less),
```
Comment (BusyJay) [inline src/util/codec/datum.rs]: Do Go has INFINITY? And does the implementation in Go behave the same as this?

Comment (qiuyesuifeng) [inline src/util/codec/datum.rs]: Yes, go has.
I see the expr like `a > 1` then the range is `(1, +inf)`, so maybe we should support it later.
@coocood Am i right?

Comment (coocood) [inline src/util/codec/datum.rs]: We have specific kind `KindMaxValue` for infinity.

**Comment by coocood** [comment]
LGTM

**Comment by ngaut** [comment]
PTAL @BusyJay

**Comment by qiuyesuifeng** [comment]
The `+inf/-inf` values are not in use, close it.

**Pull request closed without merging**